### PR TITLE
navigate to edited on merge config spec

### LINKED ISSUE #529: merging the config into the spec should navigate to /edited
Right now, we stay on the same page, which may be an example page.

### PR DESCRIPTION
**Pull request by shakti97**
Fix #529

### CODE REVIEW

File: src/components/config-editor/renderer.tsx [hunk 1]
```diff
@@ -25,6 +28,9 @@ export default class ConfigEditor extends React.PureComponent<Props> {
     if (!confirmation) {
       return;
     }
+    if (this.props.history.location.pathname.indexOf('/edited') === -1) {
```
Comment (siddhant1) [inline src/components/config-editor/renderer.tsx]: I think we can just check for '/edited' instead of linear search

**Comment by domoritz** [comment]
@shakti97 @siddhant1 can you finish this?

**Comment by shakti97** [comment]
Done with the changes recommended by @siddhant1 , @domoritz can you please check it and if any changes needed, let me know :)

**Comment by domoritz** [comment]
Thank you for the fix. I wonder whether we can not write to the history since navigating back doesn't actually restore a previous state. What do you think?

**Comment by domoritz** [comment]
Thank you for the fix. I wonder whether we can not write to the history since navigating back doesn't actually restore a previous state. What do you think?

**Pull request merged**